Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.cpp

```diff
@@ -5158,23 +5158,104 @@ namespace AssetProcessor
 
     void AssetProcessorManager::QueryAbsolutePathDependenciesRecursive(AZ::Uuid sourceUuid, SourceFilesForFingerprintingContainer& finalDependencyList, AzToolsFramework::AssetDatabase::SourceFileDependencyEntry::TypeOfDependency dependencyType)
     {
+        struct PathOrUuid
+        {
+            explicit PathOrUuid(AZ::Uuid uuid)
+                : m_uuid(uuid)
+            {
+            }
+
+            explicit PathOrUuid(AZStd::string path) : m_path(AZStd::move(path))
```

Comment:
nit: please chose a style for initializing the data member(s), either on the same line or on separate ones